### DISCUSSION TITLE
SSD Rollover

### DISCUSSION THREAD

**Discussion by James-MF-Thomas** [UltimaHoarder/UltimaScraper#752] [General]
This may have been covered already somewhere, if so just let me know. (it sounds like you reference this in the readme.)

Once one ssd/hdd has been filled or reached the volume limit via .config setting, how do you specify the second location in the .config to have the media rollover to another drive? Assuming that the script will read the metadata from both drives.  Or is this not possible?

**Comment by UltimaHoarder** [comment]
Let's say this is your download_directories
```
"download_directories": [
          "D:\\OFPanel\\Downloads",
          "E:\\OFPanel\\Downloads",
          "F:\\OFPanel\\Downloads",
          "C:\\OFPanel\\Downloads"
        ]
```

The script will look at D:\\ and check if it's full. If it isn't full, it will download stuff there.
But what happens if D:\\ if full and we try to download the same model...? Well at this time the script will end up duplicating the model folder by downloading it into E:\\ and you will have duplicate metadata.

I haven't implemented it yet, but soon you'll be able to store your metadata in a different directory other than the model folder, so dupes like this wouldn't happen.

**Reply by UltimaHoarder** [reply]
metadata_directories works now
